pocs/utils/rs232.py

```diff
@@ -142,27 +156,26 @@ def write(self, value):
 
         return response
 
-    def read(self):
-        """
-        Reads value using readline
-        If no response is given, delay and then try to read again. Fail after 10 attempts
+    def read(self, retry_limit=5, retry_delay=0.5):
```

Comment:
Do we want these params set at the class level?

Reply:
Sure, probably a good idea.